responsibleai/tests/rai_insights/test_rai_insights_validations.py

```diff
@@ -377,6 +378,52 @@ def test_classes_exceptions(self):
         assert 'The train labels and distinct values in ' + \
             'target (train data) do not match' in str(ucve.value)
 
+        X_train[TARGET] = y_train
+        X_test[TARGET] = y_test
+
+        with pytest.raises(UserConfigValidationException) as ucve:
+            RAIInsights(
+                model=model,
+                train=X_train,
+                test=X_test,
+                target_column=TARGET,
+                task_type='classification',
+                classes=[0, 1],
+                feature_metadata=FeatureMetadata(dropped_features=[
+                    "mean radius",
```

Comment:
you could just pass train.columns.tolist() here instead of the entire list of actual features here.

Reply:
updated!